src/components/Modal/Global/ConfirmModalWrapper.tsx

```diff
@@ -10,23 +12,7 @@ type ConfirmModalWrapperProps = ModalProps & Omit<ConfirmModalProps, 'onConfirm'
 // - handle closeModal inside ConfirmModal
 // - remove ConfirmModalWrapper
 
-function ConfirmModalWrapper({
```

Comment:
Why did you change the props approach in this pr? For cherry picks please always include as minimal diff as possible